test/local/email/bounce.js

```diff
@@ -257,17 +259,18 @@ describe('bounce messages', () => {
           ]
         }
       })
-      return mockedBounces(mockLog, mockDB).handleBounce(mockMsg).then(function () {
+      return mockedBounces(log, mockDB).handleBounce(mockMsg).then(function () {
         assert.equal(mockDB.emailRecord.callCount, 1)
         assert.equal(mockDB.emailRecord.args[0][0], 'test@example.com')
         assert.equal(mockDB.deleteAccount.callCount, 1)
         assert.equal(mockDB.deleteAccount.args[0][0].email, 'test@example.com')
-        assert.equal(mockLog.messages.length, 5)
-        assert.equal(mockLog.messages[3].args[0].op, 'handleBounce')
-        assert.equal(mockLog.messages[3].args[0].email, 'test@example.com')
-        assert.equal(mockLog.messages[4].args[0].op, 'databaseError')
-        assert.equal(mockLog.messages[4].args[0].email, 'test@example.com')
-        assert.equal(mockLog.messages[4].args[0].err.errno, error.ERRNO.ACCOUNT_UNKNOWN)
+        assert.equal(log.info.callCount, 2)
```

Comment:
This is much nicer now that all the various levels of logging are not mixed in to a single `messages` array!